src/app_service/service/chat/service.nim

```diff
@@ -287,6 +286,20 @@ QtObject:
         return true
     return false
 
+  proc sectionUnreadMessagesAndMentionsCount*(self: Service, communityId: string):
+      tuple[unviewedMessagesCount: int, unviewedMentionsCount: int] =
+    if communityId == "":
+      return
+
+    result.unviewedMentionsCount = 0
+    result.unviewedMessagesCount = 0
+    for chat in self.channelGroups[communityId].chats:
+      result.unviewedMentionsCount = result.unviewedMentionsCount + chat.unviewedMentionsCount
+      if chat.muted:
+        continue
+      if chat.unviewedMessagesCount > 0:
+        result.unviewedMessagesCount = result.unviewedMessagesCount + chat.unviewedMessagesCount
```

Comment:
`+=`?

Reply:
So, I was pretty sure this didn't work in Nim. And I'm right, because I tried to do it in the model and it didn't want to compile. However, in the service it worked. I have no idea why. I tried importing libs in the model and it still didn't want to work.
Weird.